addons/board/static/src/js/board_view.js

```diff
@@ -338,14 +362,19 @@ var BoardRenderer = FormRenderer.extend({
 
         // render each view
         _.each(this.actionsDescr, function (action) {
```

Comment:
I think it can be fixed too by doing (I commented the part who doesn't change):
```js
//_.each(this.actionsDescr, function (action) {
    if (!this._boardSubcontrollers.length) {
		/*self.defs.push(self._createController({
	        $node: $html.find('.oe_action[data-id=' + action.id + '] .oe_content'),
	        actionID: _.str.toNumber(action.name),
	        context: action.context,
	        domain: Domain.prototype.stringToArray(action.domain, {}),
	        viewType: action.view_mode,
	    }));*/
	} else {
		this._boardSubcontrollers.forEach(function (controller) {
            controller.appendTo($html.find('.oe_action[data-id=' + action.id + '] .oe_content'));
        });
	}
});
```

Reply:
I think you could even do something like this to reuse the old content:

```
diff --git a/addons/board/static/src/js/board_view.js b/addons/board/static/src/js/board_view.js
index ffdcfbeb7ed..7e12daebbe2 100644
--- a/addons/board/static/src/js/board_view.js
+++ b/addons/board/static/src/js/board_view.js
@@ -299,6 +299,9 @@ var BoardRenderer = FormRenderer.extend({
         // we add the o_dashboard class to the renderer's $el. This means that
         // this function has a side effect.  This is ok because we assume that
         // once we have a '<board>' tag, we are in a special dashboard mode.
+        if (this.$el.hasClass('o_dashboard')) {
+            return this.$el.children();
+        }
         this.$el.addClass('o_dashboard');
         this.trigger_up('enable_dashboard');
 
diff --git a/addons/mail/models/mail_message.py b/addons/mail/models/mail_message.py
index 6d374f7ae9c..7b59f3314e4 100644
--- a/addons/mail/models/mail_message.py
+++ b/addons/mail/models/mail_message.py
```

but for stability it seems a little more safe to recreate the controllers as they do